FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:232 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0xc00282bb 0xc0022856 0xc002aae6 0xc00210ef
Translation of call stack:
0xc00282bb: debug_panic (.../../lib/kernel/debug.c:38)
0xc0022856: lock_release (...../../threads/synch.c:234)
0xc002aae6: acquire1_thread_func (.../priority-donate-one.c:53)
0xc00210ef: kernel_thread (..../../threads/thread.c:402)
